### PR TITLE
Changelog for 0.41.0

### PR DESCRIPTION
**Pull request by seibert**

### CODE REVIEW

**Comment by seibert** [comment]
Note: Hold changelog until remaining in-flight PRs are merged and added to release notes.

File: CHANGE_LOG [hunk 1]
```diff
@@ -1,3 +1,113 @@
+Version 0.41.0
+--------------
+
+This release adds the following major features:
+
+* Official ARMv7 (Raspberry Pi 2/3/3+) support - Now building conda packages
```
Comment (stuartarchibald) [inline CHANGE_LOG]: think one of the `np.trace()` tests is failing, preventing this from shipping at present.

Comment (seibert) [reply CHANGE_LOG]: Good point.  Removed from summary list.

File: CHANGE_LOG [hunk 2]
```diff
@@ -1,3 +1,115 @@
...
+* PR #3397: Fix with-objmode warning
+* PR #3403 Fix label offset in call inline after parfor pass
+* PR #3429: Fixes raising of user defined exceptions for exec(<string>).
+* PR #3432: Fix error due to function naming in CI in py2.7 
+* PR #3444: Fixed TBB's single thread execution and test added for #3440
+* PR #3449: Allow matching non-array objects in find_callname()
+* PR #3455: Change getiter and iternext to not be pure. Resolves #3425
+* PR #3467: Make ir.UndefinedType singleton class.
+* PR #3478: Fix np.random.shuffle sideeffect
+* PR #3487: Raise unsupported for kwargs given to `print()`  5 - Ready to merge
```
Comment (stuartarchibald) [inline CHANGE_LOG]: can probably drop the GH labels like `5 - Ready to merge`?

File: CHANGE_LOG [hunk 3]
```diff
@@ -1,3 +1,115 @@
...
+* PR #3488: Remove dead script.  5 - Ready to merge
+* PR #3498: Fix stencil support for boolean as return type
+* PR #3511: Fix handling make_function literals (regression of #3414)
+
+Documentation Updates:
+
+* PR #3318: Tweak spacing to avoid search box wrapping onto second line
```
Comment (stuartarchibald) [inline CHANGE_LOG]: #3316 is missing but guess that's ok as its just noting a changelog change.

**Comment by stuartarchibald** [comment]
I've checked all the PRs noted manually against a locally generated list and they seem to correlate.

File: CHANGE_LOG (hunk 2 above)
Comment (seibert) [reply CHANGE_LOG]: Oops, copy paste fail.  Fixed now

File: CHANGE_LOG (hunk 3 above)
Comment (seibert) [reply CHANGE_LOG]: Good call.  My query to select PRs is a little more complicated with the branching

**Pull request merged**